spring-cloud-alicloud-oss/src/main/java/com/alibaba/alicloud/oss/resource/OssStorageResource.java

```diff
@@ -155,9 +156,9 @@ public InputStream getInputStream() throws IOException {
 	 * @return the bucket if it exists, or null otherwise
 	 */
 	public Bucket getBucket() {
-		return this.oss.listBuckets().stream()
-				.filter(bucket -> bucket.getName().equals(this.bucketName)).findFirst()
-				.get();
+		Optional<Bucket> value = this.oss.listBuckets().stream()
+				.filter(bucket -> bucket.getName().equals(this.bucketName)).findFirst();
+		return value.orElse(null);
```

Comment:
`return this.oss.listBuckets().stream().filter(bucket -> bucket.getName().equals(this.bucketName)).findFirst().orElse(null)` is better.

Reply:
Yes, thank for your guidance. 😀